features/main/custom_operation.feature

```diff
@@ -2,10 +2,13 @@ Feature: Custom operation
   As a client software developer
   I need to be able to create custom operations
 
+  Background:
```

Comment:
oh this is nice, does this allow to add steps to every scenario?

Reply:
Yeah.